Dockerfile

```diff
@@ -22,9 +22,9 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-# Use CentOS7
-FROM centos:7
+FROM inveniosoftware/centos7-python:3.6
```

Comment:
I noticed that there is [only one tag](https://hub.docker.com/r/inveniosoftware/centos7-python/tags) for this Docker image while there have been [several builds](https://hub.docker.com/r/inveniosoftware/centos7-python/builds) in the past six months.  Is the current `3.6` tag stable enough for production use? Shall we try to push for adopting semver2 image numbering in `inveniosoftware/centos7-python` for better safety and reproducibility?